scripts/run_metadata

```diff
@@ -0,0 +1,34 @@
+#! /bin/sh
+set -e
+set -x
+
+for test in bbbc mitocheck tubhiswt; do
```

Comment:
Tried to execute

```
[sbesson@ome-c6220-1-4 ~]$ docker run --rm -it --entrypoint=bash  -v /scratch/tmp/data:/data snoopycrimecop/ome-files-performance:master_merge_trigger
root@471c32e7da88:/git/ome-files-performance# ./scripts/run_metadata 
```

It turns out all the variable `iterations`, `datapath`, `outputpath` & `binpath` need to be re-exported.

For running a given benchmark, are we recommending to use the top-level `./run_benchmarking {tiling,pixeldata,metadata}` script or are these scripts meant to be executed as standalone?